pom.xml

```diff
@@ -451,5 +454,22 @@ Note: The maven build currently needs to be started from the javaslang root dir
                 </pluginManagement>
             </build>
         </profile>
+        <!-- A profile for running the benchmarks -->
+        <profile>
+            <id>benchmark</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>exec-maven-plugin</artifactId>
+                        <version>${maven.exec.version}</version>
+                        <configuration>
+                            <classpathScope>test</classpathScope>
+                            <mainClass>javaslang.JmhRunner</mainClass>
```

Comment:
@danieldietrich, are you sure it's working? I still couldn't get it to work.
Could you try it with skipped tests or the `test-compile` phase instead? (should take ~3 hours to complete, anything shorter is just tests):

Is this also working for you
```
mvn test-compile -pl javaslang-benchmark -P benchmark -D
```
or
```
mvn test -pl javaslang-benchmark -P benchmark -D skipTests
```
?